yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -774,3 +777,133 @@ func (yb *TargetYugabyteDB) recordEntryInDB(tx pgx.Tx, batch Batch, rowsAffected
 	}
 	return nil
 }
+
+func (yb *TargetYugabyteDB) GetDebeziumValueConverterSuite(isStreamingMode bool) map[string]func(string) (string, error) {
+	converterValueSuite := make(map[string]func(string) (string, error))
+	converterValueSuite["io.debezium.time.Date"] = func(columnValue string) (string, error) {
+		epochDays, err := strconv.ParseUint(columnValue, 10, 64)
+		fmt.Printf("epochDays: %v", epochDays)
+		if err != nil {
+			return columnValue, fmt.Errorf("parsing epoch seconds: %v", err)
+		}
+		epochSecs := epochDays * 24 * 60 * 60
+		return time.Unix(int64(epochSecs), 0).Local().Format(time.DateOnly), nil
+	}
+	converterValueSuite["io.debezium.time.Timestamp"] = func(columnValue string) (string, error) {
+		epochMilliSecs, err := strconv.ParseInt(columnValue, 10, 64)
```

Comment:
so the int 64 bit size is equivalent to long right?

Reply:
Yes